doc-Infrastructure_Migration_Solution_Guide/cfme/modules/proc_Installing_configuring_multi-cf.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// IMS_1.2/master.adoc
+// IMS_1.3/master.adoc
+[id="Installing_configuring_cf_distributed_{context}"]
+= Installing and configuring Red Hat CloudForms in a distributed environment
+
+You can install Red Hat CloudForms in a distributed environment, with CloudForms appliances on your user interface, database, and worker machines.
+
+[NOTE]
+====
+If you create an infrastructure mapping or migration plan and then remove or change a provider in CloudForms, the mapping or plan must be deleted and re-created.
+====
+
+.Procedure
+
+ifdef::rhv_1-2_vddk,rhv_1-3_vddk[]
+. Install Red Hat CloudForms on link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html/installing_red_hat_cloudforms_on_red_hat_virtualization[Red Hat Virtualization (RHV)] or link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html/installing_red_hat_cloudforms_on_vmware_vsphere/index[VMware].
```

Comment:
Query: Is the installation of CFME appliance is dependent on provider based on destination provider used in IMS as mentioned in guide
for vSphere->RHV = CFME should install on vSphere or RHV only
for vSphere->OSP = CFME should install on vSphere or OSP only

Reply:
Yes, the ifdef tags indicate which guides have the instructions (RHV users see one thing, OSP another)

Please see the doc preview in the PR description.